docker-compose.yml

```diff
@@ -0,0 +1,53 @@
+version: "3.8"
+
```

Comment:
```suggestion
```

The [version element is now obsolete according to the compose docs](https://docs.docker.com/compose/compose-file/04-version-and-name/#version-top-level-element-obsolete). It will always use the last version now.